frontend/src/components/Graph.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+    import {
+        ColorType,
+        CrosshairMode,
+        type CandlestickData,
+    } from "lightweight-charts";
+    import { Chart, CandlestickSeries } from "svelte-lightweight-charts";
+    import { currentStock } from "$lib/stores";
+    import api from "$lib/api";
+    import { onDestroy } from "svelte";
+
+    let data: CandlestickData[];
+
+    const fetchData = async () => {
+        const response = await api.price($currentStock.ticker).history();
+        data = response.map(function (price) {
+            return {
```

Comment:
This might help (found the correct types by inspecting the source):

```ts
data = response.map((price) => ({
    ...price,
    time: price.time as UTCTimestamp,
}));
```

Otherwise good job!